src/Controller/WidgetsController.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Controller;
+
+use App\Entity\Widgets;
+use App\Repository\WidgetsRepository;
```

Comment:
App\Repository\WidgetsRepository is missing and I don't find it on the file system.

```
In RegisterControllerArgumentLocatorsPass.php line 173:
Cannot determine controller argument for "App\Controller\WidgetsController::index()": the $widgetsRepository argument is type-hinted with the non-existent class or interface: "App\Repository\WidgetsRepository".
```